app/views/devise/two_factor_authentication/show.html.slim

```diff
@@ -13,15 +13,16 @@ p.mt-tiny.mb0#2fa-description
   .mb3
     = label_tag 'otp_method',
       t('devise.two_factor_authentication.otp_method.title'),
-      class: 'hide block bold'
-    label.radio.mb0.mr3
-      = f.radio_button :otp_method, 'sms', checked: true
-      span.indicator
-      = t('devise.two_factor_authentication.otp_method.sms')
-    label.radio.mb0
-      = f.radio_button :otp_method, 'voice'
-      span.indicator
-      = t('devise.two_factor_authentication.otp_method.voice')
+      class: 'hide block bold', id: 'otp-description'
+    div role="radiogroup" aria-labelledby="otp-description"
```

Comment:
should this be a `fieldset`or `legend` instead? see https://dequeuniversity.com/rules/axe/2.0/radiogroup?application=axeAPI